### PR TITLE
Add assignees for "Release" issues

### PR DESCRIPTION
**Pull request by kmkumaran**
Added few more Bot rules for "Release" specific issues.

### CODE REVIEW

**Comment by kmkumaran** [comment]
@bryanmacfarlane - Can you please review the changes done to issues-rules.yml for the bot rules.

File: issue-rules.yml [hunk 1]
```diff
@@ -59,17 +59,174 @@ rules:
   assign: ['madhurig']     
 # Area: Release
 - valueFor: '**Enter Task Name**'
-  contains: 'Azure'
```
Comment (bryanmacfarlane) [inline issue-rules.yml]: At first I wondered why just not keep the general Azure rule but I see now that you want to optimize for assign.  Nice!

Comment (bryanmacfarlane) [reply issue-rules.yml]: Also take a look at the fall back rules (text contains).  However, I think this subset of changes are good.  From here out, I think you take it on a case by case basis of "why was this issue not routed correctly"

**Comment by bryanmacfarlane** [comment]
👍

File: issue-rules.yml (hunk 1 above)
Comment (kmkumaran) [reply issue-rules.yml]: Sure. Make sense.

**Comment by kmkumaran** [comment]
Can you please explicitly approve the PR. Currently it is blocked as there is no approver.

**Comment by kmkumaran** [comment]
Thanks @bryanmacfarlane / @sachinma for the review.

**Pull request merged**